community/dash/sources

```diff
@@ -0,0 +1 @@
+http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.10.tar.gz
```

Comment:
```sh
http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.10.2.tar.gz
```

This looks to be the latest version. :+1:

Reply:
k